### DISCUSSION TITLE
Provide Obsidian API as standalone library.

### DISCUSSION THREAD

**Discussion by nathan130200** [Naamloos/Obsidian#277] [Ideas]
Hello, obsidian API is well implemented + documented, i have an suggestions to release in nuget as standalone package for who wanna create an server/client from stratch. 

For example i'm working on a raw server in 1.8.8 for minigames using pure c#

Or something like Cryengine, ProtocolBuilder, so we can manually implement & provide protocol versions to obsidian.

**Comment by Naamloos** [comment]
Hi! Apologies for the late response.

We did at some point have automated nuget releases, but we took it down since we're still in pre-release and we don't know whether this will be the final version of the API yet.

Obsidian.API _will_ eventually release on Nuget, just not right now.

Thank you for your interest in Obsidian! :)

**Reply by nathan130200** [reply]
Thanks for reply!
